content/en/docs/setup/install/standalone-operator/index.md

```diff
@@ -0,0 +1,212 @@
+    tracing                  ClusterIP      172.21.2.245     <none>          80/TCP                                                                                                                                       2m
+    zipkin                   ClusterIP      172.21.182.245   <none>          9411/TCP                                                                                                                                     2m
+    {{< /text >}}
+
+    {{< tip >}}
+    If your cluster is running in an environment that does not
+    support an external load balancer (e.g., minikube), the
+    `EXTERNAL-IP` of `istio-ingressgateway` will say
+    `<pending>`. To access the gateway, use the service's
+    `NodePort`, or use port-forwarding instead.
+    {{< /tip >}}
+
+1.  Ensure corresponding Kubernetes pods are deployed and have a `STATUS` of `Running`:
+
+    {{< text bash >}}
+    $ kubectl get pods -n istio-system
+    NAME                                                           READY   STATUS      RESTARTS   AGE
+    grafana-f8467cc6-rbjlg                                         1/1     Running     0          1m
+    istio-citadel-78df5b548f-g5cpw                                 1/1     Running     0          1m
+    istio-cleanup-secrets-release-1.1-20190308-09-16-8s2mp         0/1     Completed   0          2m
+    istio-egressgateway-78569df5c4-zwtb5                           1/1     Running     0          1m
+    istio-galley-74d5f764fc-q7nrk                                  1/1     Running     0          1m
+    istio-grafana-post-install-release-1.1-20190308-09-16-2p7m5    0/1     Completed   0          2m
+    istio-ingressgateway-7ddcfd665c-dmtqz                          1/1     Running     0          1m
+    istio-pilot-f479bbf5c-qwr28                                    2/2     Running     0          1m
+    istio-policy-6fccc5c868-xhblv                                  2/2     Running     2          1m
+    istio-security-post-install-release-1.1-20190308-09-16-bmfs4   0/1     Completed   0          2m
+    istio-sidecar-injector-78499d85b8-x44m6                        1/1     Running     0          1m
+    istio-telemetry-78b96c6cb6-ldm9q                               2/2     Running     2          1m
+    istio-tracing-69b5f778b7-s2zvw                                 1/1     Running     0          1m
+    kiali-99f7467dc-6rvwp                                          1/1     Running     0          1m
+    prometheus-67cdb66cbb-9w2hm                                    1/1     Running     0          1m
+    {{< /text >}}
+
+## Deploy your application
+
+You can now deploy your own application or one of the sample applications
+provided with the installation like [Bookinfo](/docs/examples/bookinfo/).
+
+{{< warning >}}
+The application must use either the HTTP/1.1 or HTTP/2.0 protocols for all its HTTP
+traffic; HTTP/1.0 is not supported.
```

Comment:
is this from the current eval page? If doesn't seem too relevant but if it's already there seems find the keep it

Reply:
i copied this page from the current eval page - then modified to suit the standalone operator.